### PR TITLE
GH-167 start of normalizing developer api

### PR DESCRIPTION
**Pull request by vLuckyyy**
**Please describe the changes made by this PR and why they need to be merged:**

### CODE REVIEW

**Review by Jakubk15 — APPROVED** [review]

File: eternalcombat-api/src/main/java/com/eternalcode/combat/fight/drop/DropKeepInventoryManager.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.eternalcode.combat.fight.drop;
+
+import java.util.List;
+import java.util.UUID;
+import org.bukkit.inventory.ItemStack;
+
+public interface DropKeepInventoryManager {
```
Comment (P1otrulla) [inline eternalcombat-api/src/main/java/com/eternalcode/combat/fight/drop/DropKeepInventoryManager.java:7]: rename -> Service? If you changed others Managers to services

**Review by P1otrulla — APPROVED** [review]

File: eternalcombat-api/src/main/java/com/eternalcode/combat/EternalCombatApi.java [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
+import com.eternalcode.combat.fight.tagout.FightTagOutService;
 
 public interface EternalCombatApi {
 
     FightManager getFightManager();
 
     RegionProvider getRegionProvider();
 
-    FightPearlManager getFightPearlManager();
+    FightPearlService getFightPearlManager();
```
Comment (CitralFlo) [inline eternalcombat-api/src/main/java/com/eternalcode/combat/EternalCombatApi.java:17]: You are changing the class name and then leaving the method unchanged?

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutController.java [hunk 3]
```diff
@@ -10,19 +9,17 @@
...
+    public FightTagOutController(FightTagOutService tagOutService) {
         this.tagOutService = tagOutService;
-        this.config = config;
     }
 
     @EventHandler
     void onTagOut(FightTagEvent event) {
         UUID uniqueId = event.getPlayer();
 
         if (this.tagOutService.isTaggedOut(uniqueId)) {
```
Comment (CitralFlo) [inline eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutController.java:21]: same here

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/FightTagCommand.java [hunk 4]
```diff
@@ -56,7 +56,7 @@ void tag(@Context CommandSender sender, @Arg Player target) {
         FightTagEvent event = this.fightManager.tag(targetUniqueId, time, CauseOfTag.COMMAND);
 
         if (event.isCancelled()) {
-            this.announcer.sendMessage(sender, event.getCancelMessage());
+            this.announcer.sendMessage(sender, this.config.messages.admin.adminTagOutCanceled);
```
Comment (CitralFlo) [inline eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/FightTagCommand.java:59]: what happens with this message?

**Review by CitralFlo — APPROVED** [review]

**Review by Rollczi — APPROVED** [review]

**Pull request merged**